js/components/serviceSection/serviceRenderFeatures.js

```diff
@@ -0,0 +1,35 @@
+function serviceRenderFeatures(selector, duomenys) {
+
+    if (typeof selector !== "string" || (!Array.isArray(duomenys.data))) {
+        console.error("Netinkami duomenys");
+        return false;
+    }  /* jei selektorius ne stringas ir jei Array yra ne array - blogi duomenys*/
+    
+ 
+    const DOM = document.querySelector(selector)
+    if (!DOM) {
+        console.error("Netinkama vieta HTML");
+        return false;
+    }
+
+
+
+    let HTML = '';
+    const limit = duomenys.maxLimit || duomenys.data.length; /* limit - iesko maksimalaus leistino atvaizduoti skaiciaus ir data folderyje nurodytu duomenu ilgio, jei nebutu nurotyta max skaiciua, funkcija ziuretu duomenis */
```

Comment:
Cia viskas kaip ir neblogai, tik problema islenda tada, kai tavo MaxLimit yra didesnis uz duotu duomenu kieki. Jeigu duomenu yra 3, o max limit duodamas 4 , iskart meta klaida , nes negali sugeneruot korteles,nes yra tuscia vieta

Reply:
visos validacijos bus atliekamos veliau, dabar praktiskai be ju. Kitaip tariant static